pipelines/utilities/longhorn_manifest.sh

```
@@ -1,53 +1,66 @@
-  LONGHORN_REPO_URI=${LONGHORN_REPO_URI:-"https://github.com/longhorn/longhorn.git"}
-  LONGHORN_REPO_BRANCH=${LONGHORN_REPO_BRANCH:-"master"}
-  LONGHORN_REPO_DIR="${TMPDIR}/longhorn"
+get_longhorn_repo(){
+  CHART_VERSION="${1:-$LONGHORN_REPO_BRANCH}"
 
-  CUSTOM_LONGHORN_MANAGER_IMAGE=${CUSTOM_LONGHORN_MANAGER_IMAGE:-"longhornio/longhorn-manager:master-head"}
-  CUSTOM_LONGHORN_ENGINE_IMAGE=${CUSTOM_LONGHORN_ENGINE_IMAGE:-"longhornio/longhorn-engine:master-head"}
+  # create and clean tmpdir
+  TMPDIR="/tmp/longhorn"
+  mkdir -p ${TMPDIR}
+  rm -rf "${TMPDIR}/"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)